doc/basics.md

```diff
@@ -1166,6 +1167,46 @@ std::cout << doc.find_field("k0") << std::endl; // Prints 27
 When the JSON path is the empty string (`""`) applied to a scalar document (lone string, number, Boolean or null), a SCALAR_DOCUMENT_AS_VALUE error is returned because scalar document cannot
 be represented as `value` instances. You can check that a document is a scalar with the method `scalar()`.
 
+JSON Path
+------------
+
+The simdjson library now supports a subset of [JSON Path](https://goessner.net/articles/JsonPath/) through the `at_path()` method, allowing you to reach further into the document in a single call. For now this is only supported for the On Demand approach. The subset of JSON path that is implemented is the subset that is trivially convertible into the JSON Pointer format, using `.` to access a field and `[]` to access a specific index.
```

Comment:
I recommend removing `For now this is only supported for the On Demand approach.`  You are editing the core documentation which is all about On Demand, so there is no implication that it is supported in DOM. Though the meaning of this sentence is clear to you and me, it is probably going to be confusing to some. "Why is he telling me that it is sometimes not supported?"

Reply:
I recommend documenting what happens if I use an unsupport Path expression. Maybe with a code example?

Reply:
Sure! I've added 2 examples (one where the conversion fails and returns INVALID_JSON_POINTER and another one where the conversion succeeds and at_pointer returns the error for invalid field)